docs/index.md

```diff
@@ -1,5 +1,8 @@
 ---
 title: The Sphinx Book Theme
+theme:
```

Comment:
```suggestion
theme_html:
```

(or `html_theme`)

this makes the purpose clearer, and would allow for e.g. the future addition of `theme_latex` etc

Reply:
what do you think about nested?

```
theme:
    html:
```

Reply:
ah I just realized that this might be weird to parse in Sphinx though, since the nested levels are stringified right?

Reply:
My feeling is that we don't want to constrain MyST (or Jupyter Book) on what is possible with rST. I'd be OK with telling users "if you want nested metadata in the header, then your content should be MyST". Any objection to that?

I have a slight preference for:

```yaml
theme:
    html:
        foo: bar
    latex:
		foo: bar
```

over

```yaml
theme_html:
	foo: bar
theme_latex:
	foo: bar
```

but not a strong opinion, what do you think?

Reply:
yeh, no strong preference either